package/MDAnalysis/core/groups.py

```diff
@@ -136,6 +136,28 @@ def __add__(self, other):
 
         return self.__class__(np.concatenate([self._ix, o_ix]), self._u)
 
+    def __radd__(self, other):
```

Comment:
Yes. For some reason `sum(a)` is defined as `start + a[0] + a[1] + ...` where `start` is 0 by default. The user could either pass the optional second argument to sum to specify start as an empty atomgroup of the same universe, or we can override `__radd__` so that `0 + a[0] = a[0]`. Do you think this is a worthwhile feature?